orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/cluster/WaitForClusterDisableTask.groovy

```diff
@@ -79,9 +79,14 @@ class WaitForClusterDisableTask extends AbstractWaitForClusterWideClouddriverTas
     // Even if the server group is disabled, we want to make sure instances are down
     // to prevent downstream stages (e.g. scaleDownCluster) from having to deal with disabled-but-instances-up server groups
     def targetServerGroup = serverGroup.get()
-    if (targetServerGroup.isDisabled() || stage.context.desiredPercentage) {
+    if (targetServerGroup.isDisabled() && stage.context.desiredPercentage) {
       return !waitForRequiredInstancesDownTask.hasSucceeded(stage, targetServerGroup as Map, targetServerGroup.getInstances(), interestingHealthProviderNames)
     }
+    
+    if(targetServerGroup.isDisabled()) {
```

Comment:
this is going to fail tests in `WaitForClusterDisableTaskSpec.groovy`

Reply:
Yup, based on @ajordens feedback looks like we're going to take a different approach for Kubernetes.